README.md

```diff
@@ -100,14 +99,11 @@ parameters, in which case they do not specify `oauth2` or `zcap` properties.
 
 #### Testing locally
 
-If you want to test your implementation locally, you can add a configuration
-file in the root directory of the specific test suite that you are running.
-
-```
-localImplementationsConfig.cjs
-```
+If you want to test your implementations for endpoints running locally, you can
+create a configuration file `localImplementationsConfig.cjs` in the root
```

Comment:
Does this need a `.` prefix? The text here and in the similar CCG repo https://github.com/w3c-ccg/vc-test-suite-implementations/pull/98 mention them both with and without the `.`, so we should clarify that both places--one way or the other.

Reply:
Thanks, fixed here [4057386](https://github.com/w3c/vc-test-suite-implementations/pull/10/commits/40573864081f9834fb293698b8c2600d4d1987ce).